src/vs/workbench/contrib/terminalContrib/chat/browser/media/terminalChatWidget.css

```diff
@@ -3,27 +3,71 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.terminal-inline-chat {
+.monaco-workbench .terminal-inline-chat {
 	position: absolute;
 	left: 0;
 	bottom: 0;
 	z-index: 100;
 	height: auto !important;
 }
 
-.terminal-inline-chat .inline-chat {
+.monaco-workbench .terminal-inline-chat .inline-chat {
 	margin-top: 0 !important;
+	color: inherit;
+	border-radius: unset;
+	border: unset;
+	box-shadow: unset;
+	background: var(--vscode-inlineChat-background);
```

Comment:
We might be able to do away with the unsettling  and simply remove some styles here https://github.com/microsoft/vscode/blob/f6686e3ac767891e438e906b9b83154ad880e0c6/src/vs/workbench/contrib/inlineChat/browser/media/inlineChat.css#L7

Reply:
I agree. I'd prefer to leave this to you though as I'm not that familiar with all the places inline chat is used now and would hate to break other UI.